modules/network/outputs.tf

```diff
@@ -38,8 +38,10 @@ output "pod_nsg_id" {
   value = tolist([oci_core_network_security_group.pods[*].id])
 }
 
-output "worker_nsg_id" {
-  value = oci_core_network_security_group.workers.id
+output "pod_nsg_ids" {
```

Comment:
Please add it to the `nsg_ids` instead of a separate output. All the other individual nsg ids should be marked (via comment) as deprecated.